tests/v2/test_0127-tomask-operation.py

```diff
@@ -1135,7 +1135,7 @@ def test_BitMaskedArray():
         None,
     ]
     assert (
-        ak._v2.operations.structure.to_json(array)
+        ak._v2.operaticonvertture.to_json(array)
```

Comment:
Same thing as what @agoose77 caught up above. I fixed them both in e72c41a4758c343c9023a291f58ea255f23a351b.